src/modules/client/components/customers/infos/ProfileInfo.vue

```diff
@@ -847,38 +829,21 @@ export default {
     getQuoteLink (quote) {
       return get(quote, 'drive.link') || false;
     },
-    formatSubscriptionToDownloadQuote (subscription) {
-      const estimatedWeeklyRate = this.computeWeeklyRate(subscription);
-      const nature = NATURE_OPTIONS.find(nat => nat.value === subscription.service.nature);
-
-      return {
-        serviceName: subscription.service.name,
-        serviceNature: nature ? nature.label : '',
-        unitTTCRate: subscription.unitTTCRate ? `${this.formatNumber(subscription.unitTTCRate)}€` : '',
-        weeklyVolume: subscription.estimatedWeeklyVolume,
-        weeklyRate: estimatedWeeklyRate ? `${this.formatNumber(estimatedWeeklyRate)}€` : '',
-        sundays: subscription.sundays || '',
-        evenings: subscription.evenings || '',
-      };
-    },
     async downloadQuote (doc) {
       try {
         const quoteDriveId = get(this.company, 'customersConfig.templates.quote.driveId', null);
         if (!quoteDriveId) return NotifyWarning('Template manquant');
 
-        const subscriptions = this.subscriptions.map(this.formatSubscriptionToDownloadQuote);
+        doc.subscriptions = doc.subscriptions.map(subscription => ({
+          ...subscription,
+          nature: this.subscriptions
+            .filter(s => s.service.name === subscription.serviceName)
+            .map(s => s.service.nature)[0],
+          estimatedWeeklyRate: this.computeWeeklyRate(this.subscriptions
+            .filter(s => s.service.name === subscription.serviceName)[0]),
```

Comment:
comme dans une souscription tu as plusieurs version, je pense qu'il faut que tu mettes ca dans la methode `getSubscriptionQuoteTags` pour recuperer ceux de la bonne version et non de la derniere automatiquement

Reply:
dans getSubscriptionQuoteTags ce qu'on reçoit c'est chaque souscription, exactement comme ici, le mettre dans getSubscriptionQuoteTags ni changera rien. En plus computeWeeklyRate prend une souscription en paramètre, pas une version

Reply:
ah en fait pas besoin de back pour ca mias il ne fait pas que tu fasses le map sur `this.subscriptions` mais sur `quote.subscriptions` pour avoir les infos de la souscription au moment de la génération du devis et pas les infos actuelles

tu peux faire le test suivant : 
- creer une souscrption : 10h et 1€
- genere un devis
- ajoute une version a la souscription : 20h et 2€
- telecharge le devis
-> je vois 40€ et non 20€
de plus si je supprime la sousciption je ne peux plus telecharger le devis

Reply:
hum je ne suis pas sur de comprendre ta reponse ..

en fait pas besoin de back pour ce point mais il ne fait pas que tu fasses le map sur `this.subscriptions` mais sur `quote.subscriptions` pour avoir les infos de la souscription au moment de la génération du devis et pas les infos actuelles

tu peux faire le test suivant : 
- creer une souscrption : 10h et 1€
- genere un devis
- ajoute une version a la souscription : 20h et 2€
- telecharge le devis
-> je vois 40€ et non 20€
de plus si je supprime la sousciption je ne peux plus telecharger le devis

Reply:
et du coup si tu ne te bases que sur `quote`, ce serait bien de mettre cette partie dans `getQuotetags`